src/server/graphql/mutations/changeTaskTeam.js

```diff
@@ -0,0 +1,97 @@
+import {GraphQLID, GraphQLNonNull} from 'graphql';
+import getRethink from 'server/database/rethinkDriver';
+import ChangeTaskTeamPayload from 'server/graphql/types/ChangeTaskTeamPayload';
+import {getUserId, requireTeamMember} from 'server/utils/authorization';
+import shortid from 'shortid';
+import removeEntityKeepText from 'universal/utils/draftjs/removeEntityKeepText';
+import {TASK, TASK_INVOLVES} from 'universal/utils/constants';
+import publish from 'server/utils/publish';
+
+export default {
+  type: ChangeTaskTeamPayload,
+  description: 'Update a task with a change in content, ownership, or status',
+  args: {
+    // area: {
+    //   type: AreaEnum,
+    //   description: 'The part of the site where the creation occurred'
+    // },
+    taskId: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: 'The task to change'
+    },
+    teamId: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: 'The new team to assign the task to'
+    }
+  },
+  async resolve(source, {taskId, teamId}, {authToken, dataLoader, socketId: mutatorId}) {
+    const r = getRethink();
+    const now = new Date();
+    const operationId = dataLoader.share();
+    const subOptions = {mutatorId, operationId};
+
+    // AUTH
+    const viewerId = getUserId(authToken);
+    requireTeamMember(authToken, teamId);
```

Comment:
The user should also be a team member of the team the card is _currently_ assigned to, no?

Reply:
Oh, I see this below.